master/buildbot/changes/gerritchangesource.py

```diff
@@ -22,6 +22,9 @@
 from twisted.internet.protocol import ProcessProtocol
 from twisted.python import log
 
+def defaultProjectNameTranslator(gerritProjectName):
```

Comment:
I think we should rather encourage subclassing rather than passing function. This might not be a shared opinion, as there are tons of callback configurability in buildbot. I really dislike those.